src/Microsoft.AspNetCore.Authentication.Cookies/CookieAppBuilderExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static class CookieAppBuilderExtensions
         /// </summary>
         /// <param name="app">The <see cref="IApplicationBuilder"/> to add the middleware to.</param>
         /// <returns>A reference to this instance after the operation has completed.</returns>
-        [Obsolete]
+        [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
Hmm, it's only a warning by default, you have to set IsError true
https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.iserror(v=vs.110).aspx